index.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+require 'vendor/autoload.php';
+
+$uri = parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH);
+
+$routes = [
```

Comment:
should be moved to the additional php file (configuration file), ex.: routes.php